### DISCUSSION TITLE
Can we speed up speech_to_text_eval.py?

### DISCUSSION THREAD

**Discussion by twmht** [NVIDIA-NeMo/NeMo#5362] [Q&A]
I found out the evaluation is pretty slow, only 4709 files.

the gpu usage is very low,  it seems most of the time was done on cpu. num_workers is set to 8.

any idea?

**Comment by titu1994** [comment]
There's so many variables - what model are you using, how long are your audio files, what is you batch size etc. Can't debug without that
